src/renderer/atlas/AtlasEngine.api.cpp

```diff
@@ -7,6 +7,7 @@
 #include "Backend.h"
 #include "../../buffer/out/textBuffer.hpp"
 #include "../base/FontCache.h"
+#include <cassert>

```

Comment:
Code using assert but did not include the hader cassert.

Reply:
But is that a problem? If this code ever breaks because STL fixes their includes, then can't we just fix our build once that happens? It's not a big deal in my opinion.

In either case though, I think this include belongs into the corresponding `pch.h` file.

Reply:
> But is that a problem? If this code ever breaks because STL fixes their includes, then can't we just fix our build once that happens? It's not a big deal in my opinion.
> 
> In either case though, I think this include belongs into the corresponding `pch.h` file.

It's really not a big deal, but I think it would be better to fix it.